aten/src/ATen/native/cuda/linalg/BatchLinearAlgebra.cpp

```diff
@@ -291,8 +293,7 @@ void magmaSolveBatched<c10::complex<float>>(
   AT_CUDA_CHECK(cudaGetLastError());
 }
 
-#if MAGMA_VERSION_MAJOR >= 2 && MAGMA_VERSION_MINOR >= 5 && \
-    MAGMA_VERSION_MICRO >= 4
+#if AT_MAGMA_VERSION >= 254
```

Comment:
ah yes, this must be the bug: when magma revs to a higher version micro would reset, causing this to fail